ops/testing.py

```diff
@@ -3632,8 +3655,16 @@ def send_signal(self, sig: Union[int, str], service_names: Iterable[str]):
             message = f'cannot send signal to "{first_service}": invalid signal name "{sig}"'
             raise self._api_error(500, message) from None
 
-    def get_checks(self, level=None, names=None):  # type:ignore
-        raise NotImplementedError(self.get_checks)  # type:ignore
+    def get_checks(
+        self, level: Optional[pebble.CheckLevel] = None, names: Optional[Iterable[str]] = None
+    ) -> List[pebble.CheckInfo]:
+        if names is not None:
+            names = frozenset(names)
+        return [
+            info
+            for info in self._check_infos.values()
+            if (level is None or level == info.level) and (names is None or info.name in names)
```

Comment:
Minor: I wish we could rename the local variable `info` so that it's clear it doesn't stand for e.g. logging level `INFO`.

Reply:
There was some discussion about the name in the spec process (in a review with @benhoyt and then in the last spec review meeting). `check_info` would be most accurate, but it's a bit long, and "check" is there in the event name. `check` is problematic because a `Check` class exists, and because "check" could be a verb, although you'd expect a method, not a property in that case. Containers have `get_check` and `get_checks` so this inconsistent with that, but as above, I think it's confusing that you don't get a `Check` from those. It could be `get_check_info as a method, with the name as another attribute, but that's less convenient (more explicit, but less consistent with notices).

I don't really buy the argument that someone would think it's `logging.info` or `logging.INFO`. Is there some reason to think people would expect that?

My preference is to stick with `info` but I'm open to other suggestions.

Reply:
Is all Dima is asking about the local variable name? That would be fine to change -- but I disagree it's needed here. This is just a variable local to the list comprehension, so less context seems fine to me.

But as for the `.info` property, yeah, that's already been decided in the spec review process.

Yeah, let's stick with what Tony has here.

Reply:
Ah, I missed the "local" in the comment. Indeed, probably the intention was just the local variable name. And I guess then the reason that it brings logging to mind is that there's "info.level" which seems a bit "logging level"-y.

But for the local name then I would agree with Ben's first paragraph.